README.md

```diff
@@ -11,7 +11,7 @@
 | Member (uniqname) | Role | Office Hours |
 | :---------------- | :--- | :----------- |
 | [Mosharaf Chowdhury](http://www.mosharaf.com/) (mosharaf) | Faculty | 4820 BBB. **By appointments only.**
-| [Jiachen Liu](https://websites.umich.edu/~amberljc/) (amberljc) | GSI | TBA
+| [Jiachen Liu](https://websites.umich.edu/~amberljc/) (amberljc) | GSI | 4828 BBB. F 12:30 - 1:30 PM
```

Comment:
Will this be enough room if multiple groups show up? I think using the room may be better, but up to you.

Reply:
Yea, IOE building is a bit far, I will use the corner of 4th floor (close to stairs) if too many students show up.